src/background/background.coffee

```diff
@@ -74,7 +74,7 @@ handleRequestFromContentScript = (request, sender, sendResponse) ->
     if script.indexOf('Shim') == -1
       trackController(request.host)
     chrome.tabs.executeScript(sender.tab.id, {file: script})
-    if request.host.indexOf("youtube") > -1
+    if request.host.indexOf("youtube") > -1 || request.host.indexOf("connect.monstercat") > -1
```

Comment:
Agreed. I'll merge this and look into a better solution for that hack when
I'm back from vacation next week. Thanks for the pull request. Do you work
at monstercat?

On Saturday, January 10, 2015, therealtbs <notifications@github.com
<javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:

> In src/background/background.coffee
> <https://github.com/msfeldstein/chrome-media-keys/pull/36#discussion-diff-22764519>
> :
>
> > @@ -74,7 +74,7 @@ handleRequestFromContentScript = (request, sender, sendResponse) ->
> >      if script.indexOf('Shim') == -1
> >        trackController(request.host)
> >      chrome.tabs.executeScript(sender.tab.id, {file: script})
> > -    if request.host.indexOf("youtube") > -1
> > +    if request.host.indexOf("youtube") > -1 || request.host.indexOf("connect.monstercat") > -1
>
> This is a bad hack and should be dealt with before more services rely on
> this Shim + Controller combo
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/msfeldstein/chrome-media-keys/pull/36/files#r22764519>
> .
>